homeassistant/components/sensor/usps.py

```diff
@@ -60,6 +60,7 @@ def __init__(self, session, name):
         self._name = name
         self._attributes = None
         self._state = None
+        self.update()
```

Comment:
`update` is automatically called when the second parameter to `add_devices` is `True`.